src/scales/scale.timeseries.js

```diff
@@ -0,0 +1,196 @@
+'use strict';
+
+module.exports = function(Chart) {
+
+	var helpers = Chart.helpers;
+	var timeHelpers = helpers.time;
+
+	// Default config for a timeseries scale
+	var defaultConfig = {
+		position: 'bottom',
+
+		time: {
+			parser: false, // false == a pattern string from http://momentjs.com/docs/#/parsing/string-format/ or a custom callback that converts its argument to a moment
+			format: false, // DEPRECATED false == date objects, moment object, callback or a pattern string from http://momentjs.com/docs/#/parsing/string-format/
+			unit: false, // false == automatic or override with week, month, year, etc.
+			round: false, // none, or override with week, month, year, etc.
+			minUnit: 'millisecond',
+
+			// defaults to unit's corresponding unitFormat below or override using pattern string from http://momentjs.com/docs/#/displaying/format/
+			displayFormats: {
+				millisecond: 'h:mm:ss.SSS a', // 11:20:01.123 AM,
+				second: 'h:mm:ss a', // 11:20:01 AM
+				minute: 'h:mm:ss a', // 11:20:01 AM
+				hour: 'MMM D, hA', // Sept 4, 5PM
+				day: 'll', // Sep 4 2015
+				week: 'll', // Week 46, or maybe "[W]WW - YYYY" ?
+				month: 'MMM YYYY', // Sept 2015
+				quarter: '[Q]Q - YYYY', // Q3
+				year: 'YYYY' // 2015
+			},
+		},
+		ticks: {
+			autoSkip: true
```

Comment:
@IlyaBeliaev comment from @simonbrunel :

> I see you turn `autoSkip` back to true for the `timeseries`? actually, that's my (hopefully) last issue: since the `time` scale doesn't support `autoSkip`, it's `false` by default, so can't have the same default as your PR (`true`)

I wasn't sure why you needed to do this as I had it set to `false` in the original code: https://github.com/chartjs/chartjs-chart-financial/blob/master/src/scale.timeseries.js#L85

Is this something you can change back to `false`?